CMakeLists.txt

```diff
@@ -4,7 +4,10 @@
 cmake_minimum_required(VERSION 2.8.11)
 project(azure_iot_sdks)
 
-set(IOT_SDK_VERSION 1.1.23)
```

Comment:
This may require a release script update.  Please follow up on this so it doesn't get found at the last moment.